package.json

```diff
@@ -1,36 +1,35 @@
 {
   "name": "generator-videojs-plugin",
-  "version": "0.1.2",
-  "description": "Yeoman generator for video.js plugins",
-  "license": "MIT",
-  "main": "app/index.js",
-  "repository": "dmlap/generator-videojs-plugin",
+  "version": "0.0.0",
+  "description": "A Yeoman generator for VideoJS plugins",
+  "license": "Apache 2.0",
+  "main": "generators/app/index.js",
+  "repository": "videojs/generator-videojs-plugin",
   "author": {
-    "name": "David LaPalomento",
-    "email": "",
-    "url": "https://github.com/dmlap"
-  },
-  "engines": {
-    "node": ">=0.10.0"
+    "name": "VideoJS",
+    "url": "https://github.com/videojs"
   },
   "scripts": {
     "test": "mocha"
   },
   "files": [
-    "app"
+    "generators"
   ],
   "keywords": [
     "yeoman-generator"
   ],
   "dependencies": {
-    "chalk": "~0.4.0",
-    "underscore.string": "^2.3.3",
-    "yeoman-generator": "~0.16.0"
+    "chalk": "^1.0.0",
+    "lodash": "^3.10.1",
+    "yeoman-generator": "^0.20.0",
+    "yosay": "^1.0.2"
   },
   "devDependencies": {
-    "mocha": "*"
-  },
-  "peerDependencies": {
-    "yo": ">=1.0.0"
+    "karma-chrome-launcher": "^0.2.1",
+    "karma-firefox-launcher": "^0.1.6",
+    "karma-ie-launcher": "^0.2.0",
+    "karma-qunit": "^0.1.8",
```

Comment:
Is this to test that our generator is working and that the generated tests are working?